wafer/pages/views.py

```diff
@@ -49,3 +52,10 @@ def slug(request, url):
         return EditPage.as_view()(request, pk=page.id)
 
     return ShowPage.as_view()(request, pk=page.id)
+
+
+class PageViewSet(viewsets.ModelViewSet):
+    """API endpoint for users."""
+    queryset = Page.objects.all()
+    serializer_class = PageSerializer
+    permission_classes = (DjangoModelPermissions, )
```

Comment:
Do we perhaps want to use `DjangoModelPermissionsOrAnonReadOnly` since anonymous users can read all pages from the site already?